### ISSUE TITLE
Error: The method 'OpacitySliderTrack.paint' doesn't have the named parameter 'secondaryOffset' of overridden method 'SliderTrackShape.paint'.

### ISSUE THREAD

**Issue by RA1NO3O** [bytedance/flutter_ume#101]
## Steps to Reproduce 复现步骤
flutter run
## Error log 错误日志

``` bash
./../../.pub-cache/hosted/pub.flutter-io.cn/flutter_ume_kit_ui-1.0.0/lib/third_party/cyclop/lib/src/widgets/opacity/opacity_slider_track.dart:20:8: Error: The method 'OpacitySliderTrack.paint' doesn't have the named parameter 'secondaryOffset' of overridden method 'SliderTrackShape.paint'.
```

## Environmental info 环境信息
Flutter SDK: 3.7.0
### Flutter doctor

``` bash
[✓] Flutter (Channel stable, 3.7.0, on macOS 13.2 22D49 darwin-arm64, locale
    zh-Hans-CN)
    • Flutter version 3.7.0 on channel stable at /Users/ra1no3o/Library/Flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b06b8b2710 (3 weeks ago), 2023-01-23 16:55:55 -0800
    • Engine revision b24591ed32
    • Dart version 2.19.0
    • DevTools version 2.20.1
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
    • Android SDK at /Users/ra1no3o/Library/Android/sdk
    • Platform android-33, build-tools 33.0.1
    • Java binary at:
      /Library/Java/JavaVirtualMachines/jdk-19.jdk/Contents/Home/bin/java
    • Java version Java(TM) SE Runtime Environment (build 19.0.2+7-44)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14C18
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] Connected device (3 available)
    • iPhone 14 Pro Max (mobile) • 756CBC52-92E2-428F-B301-6B948AA535A2 • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-16-2 (simulator)
    • macOS (desktop)            • macos                                •
      darwin-arm64   • macOS 13.2 22D49 darwin-arm64
    • Chrome (web)               • chrome                               •
      web-javascript • Google Chrome 109.0.5414.119

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```

### UME and kits version UME 及插件包版本

``` bash
# UME
  flutter_ume: 1.0.2+1
  flutter_ume_kit_ui: 1.0.0
  flutter_ume_kit_device: 1.0.0
  flutter_ume_kit_perf: 1.0.0
  flutter_ume_kit_show_code: 1.0.0
  flutter_ume_kit_console: 1.0.0
  flutter_ume_kit_dio: 1.0.0
```

**Comment by tongxyj** [comment]
不引入flutter_ume_kit_ui: 1.0.0库可以运行

**Comment by talisk** [comment]
Fixed with flutter_ume_kit_ui: 1.1.0
cc @tongxyj @RA1NO3O
